src/Admin/App.js

```diff
@@ -233,9 +218,8 @@ class App extends React.Component {
               )
             }}
           />
-          {shouldShowTerms && <Terms close={() => this.hideTermsAndConditions()} />}
           {innerContent}
-          <FooterContainer feedbackEmail={config.mailTo} prisonStaffHubUrl={config.prisonStaffHubUrl} />
+          <FooterContainer supportUrl={config.supportUrl} prisonStaffHubUrl={`${config.dpsEndpointUrl}/`} />
```

Comment:
note `mailTo` now no longer needed and shared components expects a `/` at the end of the url :angry: